core/src/main/java/jenkins/model/lazy/AbstractLazyLoadRunMap.java

```diff
@@ -292,6 +293,18 @@ public R get(int n) {
     }
 
     /**
+     * Checks if the disk index contains the specified build number.
```

Comment:
I would suggest avoiding the term “disk” in APIs. It is kind of anachronistic, is it not? Perhaps just `buildExists`?